translations/fr.php

```diff
@@ -1,15 +1,15 @@
 <?php
 
 $timeAgoStrings = array(
-  'aboutOneDay' => "depuis un jour",
-  'aboutOneHour' => "depuis à peu près une heure",
-  'aboutOneMonth' => "depuis à peu près un mois",
-  'aboutOneYear' => "depuis à peu près une année",
-  'days' => "depuis %s jours",
-  'hours' => "depuis %s heures",
-  'lessThanAMinute' => "depuis moins d'une minute",
-  'lessThanOneHour' => "depuis %s minutes",
-  'months' => "depuis %s mois",
-  'oneMinute' => "depuis une minute",
-  'years' => "depuis plus de %s années"
+  'aboutOneDay' => "il y a un jour",
+  'aboutOneHour' => "il y a une heure",
+  'aboutOneMonth' => "il y a un mois",
+  'aboutOneYear' => "il y a un an",
+  'days' => "il y a %s jours",
+  'hours' => "il y a %s heures",
+  'lessThanAMinute' => "il y a moins d'une minute",
+  'lessThanOneHour' => "il y a %s minutes",
```

Comment:
```il y a moins d'une heure```

Reply:
The english translation uses `%s minutes ago`. While the key is misleading, from my understanding it would mean to go from `il y a une minute` straight to `il y a moins d'une heure`.